flow/Arena.cpp

```diff
@@ -143,7 +143,8 @@ size_t Arena::getSize(FastInaccurateEstimate fastInaccurateEstimate) const {
 		if (fastInaccurateEstimate) {
 			result = impl->estimatedTotalSize();
 		} else {
-			result = impl->totalSize();
+			std::unordered_set<ArenaBlock*> visited;
+			result = impl->totalSize(visited);
```

Comment:
You can move `visited` inside `totalSize()`, thus avoiding changing the function declaration.

Reply:
changed to pass by reference as we discussed offline